content/en/docs/creating-addons/testing/install-specific-version-of-addon.md

```diff
@@ -0,0 +1,191 @@
+---
+title: Installing a specific version of an add-on in a staging environment
```

Comment:
I know "add-on" is the right way, but we use "Addon" everywhere, mostly not to have an extra character in project names and source code... I'd suggest we keep it consistent.